openwisp_controller/config/admin.py

```diff
@@ -918,6 +929,14 @@ def get_extra_context(self, pk=None):
                 ),
             }
         )
+        # passing the whois details to the context to avoid
+        # the need to make an additional request in the js
+        if (
+            pk
+            and app_settings.WHOIS_CONFIGURED
+            and (data := json.dumps(get_whois_info(pk)))
+        ):
```

Comment:
In `get_whois_info` function we are already checking for whois configured flag. do we need to check this flag twice, can we reduce number of checks?

Also add a check for pk in that function only?